### DISCUSSION TITLE
Increasing footnote font size

### DISCUSSION THREAD

**Discussion by seewoo5** [mmistakes/minimal-mistakes#4871] [Q&A] [answered]
How can I increase the font size of footnotes? For now, it appears as really small. For example, here's [my blog post](https://seewoo5.github.io/jekyll/update/2022/09/21/infinitude-of-primes-with-ABC-conjecture.html) and you can find the tiny footnote.

**Comment by iBug** [comment]
This is not the theme's default setup, so you must have customized something for it to change. Please share your source code or we can't help.

**Reply by seewoo5** [reply]
Thanks. Here's a repository to my website (including blogs): https://github.com/seewoo5/seewoo5.github.io All the markdown files for the blog posts are under `_posts` directory. Here's an example code how I add footnotes:

```
---
layout: posts
title:  "Title"
date:   1990-01-01
categories: jekyll update
tags: math
---

Here's a text with a footnote.[^1]
Here's a text with another footnote.[^2]

[^1] Here's a footnote.
[^2] Here's second footnote.
```

**Comment by iBug** [comment, marked as answer]
Your site is missing these CSS rules for `.page__content { li, dl, p }` and it's falling back to the `.footnote { ol, li, p }`, which cause the font size to be shrunk three times.

![image](https://github.com/mmistakes/minimal-mistakes/assets/7273074/c1c25393-fb8a-474f-b929-2dc65bbfdba2)

If you don't know how to debug CSS, my suggestion is to try again with the remote theme method instead of forking with all files.

**Reply by seewoo5** [reply]
Thanks! I fixed it by changing `font-size` of `.footnotes` in `_utilities.scss`.

**Reply by msk-nightly** [reply]
I was looking to change the footnote colour because it's really light and I found it in `_utilities.scss`. The reply helped. Thank you @seewoo5
